entity/templates/src/main/java/package/domain/_Msc.java

```diff
@@ -0,0 +1,128 @@
+package <%=packageName%>.domain;
```

Comment:
MSC - it is Mapped Super Class which can be a base "@MappedSuperclass" for other entities. When entity inherite from MSC, generator collect fields from it and add to table and  js-app.